### PR TITLE
More fair bot message

### PR DESCRIPTION
**Pull request by sapegin**
After discussion with @sokra.

**What kind of change does this PR introduce?**

Bot tweak.

**Did you add tests for your changes?**

No.

**If relevant, link to documentation update:**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

### CODE REVIEW

File: open-bot.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ rules:
       identifier: "ci-result"
       readd: true
       message: |-
-        The most important CI builds succeeded. Great work so far.
+        Thank you for your pull request! We’ll review it soon.
```
Comment (sokra) [inline open-bot.yaml]: This message miss the info about the CI success.

Comment (sapegin) [inline open-bot.yaml]: I’m not really sure is useful when CI is OK but can add it back if you think we need it ;-)

> Thank you for your pull request! The most important CI builds succeeded, we’ll review the pull request soon.

Good enough?

**Comment by sapegin** [comment]
@webpack-bot What do you think about the changes? ;-)

**Comment by sokra** [comment]
Thanks

**Pull request merged**